ch05/07_gpt_to_llama/README.md

```
@@ -8,4 +8,188 @@ This folder contains code for converting the GPT implementation from chapter 4 a
-<img src="https://sebastianraschka.com/images/LLMs-from-scratch-images/bonus/gpt-to-llama/gpt-and-all-llamas.webp">
\ No newline at end of file
+<img src="https://sebastianraschka.com/images/LLMs-from-scratch-images/bonus/gpt-to-llama/gpt-and-all-llamas.webp">
+
+
+&nbsp;
+### Using Llama 3.2 via the `llms-from-scratch` package
+
+For an easy way to use the Llama 3.2 1B and 3B models, you can also use the `llms-from-scratch` PyPI package based on the source code in this repository at [pkg/llms_from_scratch](../../pkg/llms_from_scratch).
+
+&nbsp;
+##### 1) Installation
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h4; Actual: h5

(MD001, heading-increment)